tests/test_cluster.py

```diff
@@ -1,3 +1,5 @@
+import math
+
```

Comment:
Is this necessary?

Reply:
Nope, my mistake.

Reply:
I use it to find particle - center distances in one of the tests.

Reply:
Please use numpy when possible to find distances instead of using the `math` library. If you need help with this just let me know.